bundles/org.openhab.binding.modbus.kermi/src/main/resources/OH-INF/i18n/kermi_de.properties

```diff
@@ -0,0 +1,81 @@
+# thing types
```

Comment:
This file should be deleted, translations are made by crowdin. Only default translation should be provided by github.